sdks/python/apache_beam/pvalue.py

```diff
@@ -413,7 +413,16 @@ def __init__(self, side_input_data):
 
   @staticmethod
   def _from_runtime_iterable(it, options):
```

Comment:
I'm not very familiar with this module; what's the rationale of putting the view construction logic here instead of the `view_fn` in `AsMultiMap._side_input_data()`?

Reply:
`view_fn` in `AsMultiMap._side_input_data` is correctly called with a Mapping for portable runners. `_from_runtime_iterable` is only used for legacy-style runners where side inputs are always iterables.

Reply:
OK, thanks!